examples/graphite.js

```diff
@@ -8,8 +9,25 @@ const graphiteReporter = new GraphiteReporter({
   host: graphiteHost,
   port: graphitePort,
   prefix: spacePrefix,
+  defaultTags: { defaultTag: 'defaultValue' },
```

Comment:
why not call it just "tags" ?

Reply:
my thinking is to make it more explicit - i.e. inform the user that these tags will be added on each metric, but can be overridden by a specific report

Reply:
done :)